test/python/visualization/test_circuit_drawer.py

```diff
@@ -239,6 +238,15 @@ def test_reverse_bits(self):
         result = visualization.circuit_drawer(circuit, output="text", reverse_bits=True)
         self.assertEqual(result.__str__(), expected)
 
+    def test_justify_warning(self):
```

Comment:
Thanks for adding a test! If you take the above recommendation to deprecate setting an invalid value, then it'd be great to test here that every one of the public interfaces is warning correctly.

Reply:
No problem! I'll do that indeed! 🙌